docs/release/release-tasks.md

```diff
@@ -318,6 +318,19 @@ The goal of this task to make the book for the current release available under e
 3. Update references in introduction.md only on the main branch (drop unsupported versions, add the new release version).
    <br>Prior art: [Add release 1.2 book link](https://github.com/kubernetes-sigs/cluster-api/pull/6697)
 
+#### Generate weekly PR updates to post in Slack
+The goal of this task is to keep the CAPI community updated on recent PRs that have been merged. This is done by using the weekly update tool in `hack/tools/release/weekly/main.go`. Here is how to use it:
+1. Checkout the latest commit on the release branch, e.g. `release-1.4`, or the main branch if the release branch doesn't yet exist (e.g. beta release).
+2. Build the release weekly update tools binary.
+   ```bash
+   make release-weekly-update
```

Comment:
```suggestion
   make release-weekly-update-tool
```

Reply:
Sorry for missing this and thanks for catching it!